gpflow/optimizers/scipy.py

```diff
@@ -8,7 +8,7 @@
 __all__ = ["Scipy"]
 
 Loss = tf.Tensor
-Variables = Tuple[tf.Variable]
+Variables = Iterable[tf.Variable]
```

Comment:
does the ordering matter for the variables? does it need to be preserved in any way?

Reply:
No. That's why it's turned into a tuple inside the minimize()